src/MagnumPlugins/WebPImporter/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+#
+#   This file is part of Magnum.
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+#               2020, 2021, 2022 Vladimír Vondruš <mosra@centrum.cz>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+
+find_package(Magnum REQUIRED Trade)
+find_package(WebP REQUIRED)
+
+if(BUILD_PLUGINS_STATIC AND NOT DEFINED MAGNUM_WEBPIMPORTER_BUILD_STATIC)
+    set(MAGNUM_WEBPIMPORTER_BUILD_STATIC 1)
+endif()
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
+               ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
+
+# WebPImporter plugin
+add_plugin(WebPImporter
+    "${MAGNUM_PLUGINS_IMPORTER_DEBUG_BINARY_INSTALL_DIR};${MAGNUM_PLUGINS_IMPORTER_DEBUG_LIBRARY_INSTALL_DIR}"
+    "${MAGNUM_PLUGINS_IMPORTER_RELEASE_BINARY_INSTALL_DIR};${MAGNUM_PLUGINS_IMPORTER_RELEASE_LIBRARY_INSTALL_DIR}"
+    WebPImporter.conf
+    WebPImporter.cpp
+    WebPImporter.h)
+if(MAGNUM_WEBPIMPORTER_BUILD_STATIC AND BUILD_STATIC_PIC)
+    set_target_properties(WebPImporter PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endif()
+target_include_directories(WebPImporter
+    PUBLIC
+        ${PROJECT_SOURCE_DIR}/src
+        ${PROJECT_BINARY_DIR}/src
+    PRIVATE
+        ${WebP_INCLUDE_DIRS})
+target_link_libraries(WebPImporter PUBLIC
+    Magnum::Trade
+    ${WebP_LIBRARIES})
```

Comment:
You can use `WebP::libwebp` instead, which would link the library and include the header automatically. You can remove line 51 (and the "PRIVATE" there) then.